lib/cisco_node_utils/cmd_ref/vpc.yaml

```diff
@@ -194,16 +194,16 @@ system_mac:
   set_value: "<state> system-mac <mac_addr>"
   default_value: ''
 
-system_priority: 
+system_priority:
```

Comment:
Agreed, but beyond the scope of this PR.  I will open up a user story for this.  I have not added a step to Travis-CI so what are your thoughts... would you suggest that I add another line like this?
```
script:
 - bundle exec rake
 - ./scripts/check_ws_yaml.sh
```
Where the shell script has the logic to report trailing ws issues, rr is there another way to do this?